piicatcher/scanner.py

```diff
@@ -91,8 +93,9 @@ class ColumnNameScanner(Scanner):
 
     def scan(self, text):
         types = set()
+        ignore_regex = re.compile(self.excluded_column[0], re.IGNORECASE)
```

Comment:
I think you can do this in __init__ only. dont have to do it here

Reply:
Done.